cmd/recommend.go

```diff
@@ -29,6 +29,7 @@ func init() {
 	recommendCmd.Flags().StringSliceVarP(&recommendOptions.Images, "image", "i", []string{}, "Container image list (comma separated)")
 	recommendCmd.Flags().StringSliceVarP(&recommendOptions.Labels, "labels", "l", []string{}, "User defined labels for policy (comma separated)")
 	recommendCmd.Flags().StringVarP(&recommendOptions.Namespace, "namespace", "n", "", "User defined namespace value for policies")
+	recommendCmd.Flags().BoolVar(&recommendOptions.Update, "update", false, "Updates the local cache of policy-templates ($HOME/.cache/karmor)")
```

Comment:
`--update` should not be a flag in `karmor recommend`. It should be a sub-command like `karmor recommend update`.